make.py

```diff
@@ -61,15 +61,14 @@ def _print_usage():
 
 
 if __name__ == '__main__':
-    avail_cmds = dict(clean=clean, dist=dist, docs=docs)
+    avail_cmds = {clean=clean, dist=dist, docs=docs}
     try:
         command = avail_cmds[sys.argv[1]]
     except IndexError:
         # Invalid number of arguments, just print help
         _print_usage()
     except KeyError:
         print('Unknown command:', sys.argv[1])
```

Comment:
Removing the extra line hurts the readability of the output since the newline separates the error message from the usage information:
```console
$ python make.py badcommand
Unknown command: badcommand

Usage: make.py <command>
  where commands are: clean, dist, docs

clean    Clean up all build & test artifacts, and generated documentation.
dist     Create all files to distribute Pyglet
docs     Generate documentation
```
```suggestion
        print(f"Unknown command: {sys.argv[1]}\n")
```